src/Core/Migration/V6_7/Migration1765376847SetDefaultSystemConfigLoadPreviewsOnSearch.php

```diff
@@ -0,0 +1,31 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\Migration\V6_7;
+
+use Doctrine\DBAL\Connection;
+use Shopware\Core\Defaults;
+use Shopware\Core\Framework\Log\Package;
+use Shopware\Core\Framework\Migration\MigrationStep;
+use Shopware\Core\Framework\Uuid\Uuid;
+
+/**
+ * @internal
+ */
+#[Package('framework')]
+class Migration1765376847SetDefaultSystemConfigLoadPreviewsOnSearch extends MigrationStep
+{
+    public function getCreationTimestamp(): int
+    {
+        return 1765376847;
+    }
+
+    public function update(Connection $connection): void
+    {
+        $connection->insert('system_config', [
```

Comment:
Should we check if the `configuration_key` existed before inserting?

Reply:
Added a check